recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conan.tools.cmake import CMake, CMakeToolchain
```

Comment:
I don't know, is `CMakeToolchain` mature enough to be used in CCI?

Reply:
I don't know, that is for someone else to decide. We use it internally for ~25 packages, so it is what I'm most familiar with at this point.